artemis-features/src/main/resources/artemis.xml

```diff
@@ -32,21 +32,23 @@ under the License.
            MAPPED: mmap files
            NIO: Plain Java Files
        -->
-      <journal-type>ASYNCIO</journal-type>
+      <journal-type>NIO</journal-type>
 
-      <paging-directory>./data/artemis/paging</paging-directory>
+      <paging-directory>./paging</paging-directory>
```

Comment:
Why change this?

Reply:
root directory is data/artemis/<brokername>. This avoid data/artemis to be used unnecessary twice.